FluentAvalonia/Styling/ControlThemes/BasicControls/ColorPicker/ColorView.axaml

```diff
@@ -0,0 +1,625 @@
+           to see over top of the default control/window background. Since palettes do not usually have transparency
+           it is considered better to disable this functionality rather than work around it with a multi-value
+           converter. This should be revisited if Avalonia supports the above code in the future. -->
+      <Setter Property="Stroke" Value="{Binding Converter={StaticResource ContrastBrushConverter}}" />
+      <Setter Property="Opacity" Value="1" />
+    </Style>
+  </ControlTheme>
+
+  <ControlTheme x:Key="ColorViewColorModelRadioButtonTheme"
+                TargetType="RadioButton">
+    <Setter Property="Background" Value="{DynamicResource ToggleButtonBackground}" />
+    <Setter Property="Foreground" Value="{DynamicResource ToggleButtonForeground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource ToggleButtonBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource ToggleButtonBorderThemeThickness}" />
+    <Setter Property="Padding" Value="{StaticResource ButtonPadding}" />
+    <Setter Property="HorizontalAlignment" Value="Stretch" />
+    <Setter Property="VerticalAlignment" Value="Center" />
+    <Setter Property="FontFamily" Value="{DynamicResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontWeight" Value="Normal" />
+    <Setter Property="FontSize" Value="{DynamicResource ControlContentThemeFontSize}" />
+    <Setter Property="HorizontalContentAlignment" Value="Center" />
+    <Setter Property="VerticalContentAlignment" Value="Center" />
+    <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <ContentPresenter Name="PART_ContentPresenter"
+                          Background="{TemplateBinding Background}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}"
+                          Content="{TemplateBinding Content}"
+                          ContentTemplate="{TemplateBinding ContentTemplate}"
+                          CornerRadius="{TemplateBinding CornerRadius}"
+                          Padding="{TemplateBinding Padding}"
+                          HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                          VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}" />
+      </ControlTemplate>
+    </Setter>
+
+    <Style Selector="^:pointerover /template/ ContentPresenter#PART_ContentPresenter">
+      <Setter Property="Background" Value="{DynamicResource ToggleButtonBackgroundPointerOver}" />
+      <Setter Property="BorderBrush" Value="{DynamicResource ToggleButtonBorderBrushPointerOver}" />
+      <Setter Property="TextBlock.Foreground" Value="{DynamicResource ToggleButtonForegroundPointerOver}" />
```

Comment:
`Foreground` here and below

Reply:
Wow, wonder how those slipped through. They should have been caught with TextElement or the ContentPresenter PRs... so good catch!

I'll update code within a few hours.

Reply:
@amwx OK, I fix it and merged in the latest updates from your preview1 branch.

Note that the TextBlock.Foregound actually was there because that theme was copy-pasted from Fluent Avalonia's ToggleButton. ToggleButton theme (and likely many more) have this issue. A quick search for `TextBlock.Foreground` returned many results. 

I can help you update all those later if you want. Several pointing to ContentPresenter can just be Foreground. All others should be `TextElement.Foreground`.

Reply:
Wow, yea there's quite a few I missed. Clearly this is going to be a hard habit to break 😂. Thanks for checking.